short/index.html

```diff
@@ -357,27 +374,16 @@
 
 *   Write **lots** of tests for scientific core of the code
 *   Start with many fewer tests for other project components
-    *   command-line interface
-    *   file I/O
-    *   other "boring" and "unimportant" code
 *   When bugs are encountered:
+    *   write new regression tests specific to that bug  <!-- #tdd #ftw -->
     *   fix the observed bug
-    *   write new regression tests specific to that bug
```

Comment:
In my LBL talk, somebody reminded me that this was a great place to put TDD into practice. Duh!